src/Client/components/submission/CodeEditor.tsx

```
@@ -125,98 +140,171 @@ export default function CodeEditor({
+                                        const isSelected =
+                                            selectedTestCase === index
+                                                ? "ring-2 ring-blue-500"
+                                                : "";
+                                        return (
+                                            <button
+                                                key={index}
+                                                onClick={() =>
+                                                    setSelectedTestCase(index)
+                                                }
+                                                className={`w-8 h-8 rounded-lg text-white flex items-center justify-center ${getStatusColor()} ${isSelected}`}
+                                            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)